pom.xml

```diff
@@ -1104,17 +1103,19 @@ under the License.
 			</properties>
 
 			<build>
-				<pluginManagement>
-					<plugins>
-						<plugin>
-							<groupId>org.apache.maven.plugins</groupId>
-							<artifactId>maven-surefire-plugin</artifactId>
-							<configuration>
-								<excludedGroups>org.apache.flink.testutils.junit.FailsOnJava17</excludedGroups>
-							</configuration>
-						</plugin>
-					</plugins>
-				</pluginManagement>
+				<plugins>
```

Comment:
can we keep it in the plugin management? A profile like this shouldn't be _enabling_ a plugin.

Reply:
Yikes, I did a mistake when initializing the default variable [in this PR](https://github.com/apache/flink/pull/23970/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R1789). This change required for each profile to have the plugin enabled. I fixed that now :+1: